crates/store/src/key/alias.rs

```diff
@@ -0,0 +1,70 @@
+use std::convert::Infallible;
+
+use borsh::{BorshDeserialize, BorshSerialize};
+use calimero_primitives::alias::Alias;
+use generic_array::sequence::Concat;
+use generic_array::typenum::{U1, U32, U400};
+use generic_array::GenericArray;
+
+use super::component::KeyComponent;
+use super::{AsKeyParts, FromKeyParts, Key};
+use crate::db::Column;
+
+#[derive(BorshDeserialize, BorshSerialize, Clone, Copy, Debug, PartialEq, Eq)]
+#[repr(u8)]
+pub enum Kind {
+    Context,
+    Identity,
+    Application,
+}
```

Comment:
also, if you're ever going to have `#[repr(<int>)]`, add enum discriminants to the variants

```suggestion
#[repr(u8)]
pub enum Kind {
    Context = 0,
    Identity = 1,
    Application = 2,
}
```